.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: true
```

Comment:
What does this mean? Does this mean we can still open issues without the template?

Reply:
yes for issues that don't align with the template